src/plotter_disk.hpp

```diff
@@ -398,11 +420,7 @@ class DiskPlotter {
             }
 
             if (!bRenamed) {
-#ifdef _WIN32
-                Sleep(5 * 60000);
-#else
-                sleep(5 * 60);
-#endif
+                Util::sleep_seconds(5 * 60);
```

Comment:
this could just be `std::this_thread::sleep_for(5min);`

for `min` you'd need `using namespace std::chrono_literals;` as well

Reply:
Thanks! Fixed.